tdrs-backend/tdpservice/settings/common.py

```diff
@@ -421,3 +422,14 @@ class Common(Configuration):
     CELERY_TASK_SERIALIZER = 'json'
     CELERY_RESULT_SERIALIZER = 'json'
     CELERY_TIMEZONE = 'UTC'
+    CELERY_BEAT_SCHEDULE = {
+        'name': {
+            'task': 'tdpservice.scheduling.tasks.nightly_postgres',
+            'schedule': 10.0, # crontab(minute='*', hour='*'),
```

Comment:
```suggestion
            'schedule': crontab(minute='*', hour='0'),
```
The 10 second schedule was just for testing. Will plan to set this to run at midnight per the above suggestion once this is merge-ready.

Reply:
Do we need to account for UTC time zone vs Eastern?